pkg/module/file/file-example-cfg.yml

```diff
@@ -0,0 +1,63 @@
+# Example configuration for the file module
+# This file demonstrates various use cases for file transfers between client and dutagent
+# $WD is a placeholder of the working directory on both the client and dutagent
+
+devices:
+  example-device:
+    cmds:
+      # Upload any file - preserves directory structure
+      upload:
+        desc: "Upload file to device"
+        modules:
+          - module: file
+            main: true
+            options:
+              operation: "upload"
+        # Usage:
+        #   dutctl example-device upload ./build/firmware.bin
+        #   → uploads to $WD/build/firmware.bin (preserves structure)
+        #
+        #   dutctl example-device upload ./firmware.bin:/tmp/firmware.bin
```

Comment:
I thought, we would restrict absolute paths if not set via dest/src in config?

Reply:
Yes the comment was invalid